.gitignore

```diff
@@ -0,0 +1,33 @@
+# Default .gitignore for Ruby
+*.gem
+*.rbc
+.bundle
+.config
+coverage
+InstalledFiles
+lib/bundler/man
+pkg
+rdoc
+spec/reports
+test/tmp
+test/version_tmp
+tmp
+Gemfile.lock
+
+# YARD artifacts
+.yardoc
+_yardoc
+doc/
+
+# Vim
+*.swp
+
+# OS X
+.DS_Store
+
+# Puppet
+coverage/
+spec/fixtures/modules/*
+spec/fixtures/manifests/*
+# needed to spec test alternative templates
+!spec/fixtures/modules/my_rngd
```

Comment:
I would prefer if we do not parameterize the path to the config template itself. Our templates should be dynamic enough so this is not needed or we accept a patch as opposed to someone just using something different.